build.sbt

```diff
@@ -21,7 +21,7 @@ lazy val compilerOptions = Seq(
   "-Xfuture"
 )
 
-lazy val catsVersion = "0.6.1"
+lazy val catsVersion = "0.7.0-SNAPSHOT"
```

Comment:
The two releases will be coordinated—this PR won't be merged until the Cats release is out.

(I mention that in the last paragraph of the note, but I guess I should add it to the title as well. :smile:)